### ISSUE TITLE
FR: allow multiple templates

### ISSUE THREAD

**Issue by dbobak** [omnivore-app/obsidian-omnivore#36]
It would be useful to have a possibility to use multiple templates for importing from Omnivore to Obsidian. For example, one for syncing only highlights, another one for importing full article texts. Use of each of the template would be specified by filters/custom queries.

**Comment by jacksonh** [comment]
Just fleshing the idea out here: would it be something like "If i add the label `Obsdian/Reading` to an article we use this template to import the full content, otherwise use this template to import just the highlights" ?

**Comment by dbobak** [comment]
I would like to have the possibility to import all of the highlights and, for some articles, also import full text, as a separate note. So perhaps the best option would be:
- if article has label #Obsidian/Reading -> use template1
- if article has label #Obsidian/Highlights -> use template2
- if article has label #Obsidian/Important -> use template3
[and so on]
- otherwise -> use template0

Articles with multiple labels should be imported multiple times, using corresponding templates.

**Comment by ronny3** [comment]
I agree that some logic is needed, as the current template language is logic-less. And in addition the label feels like the proper place to attach that parsing-logic, as the alternative would be a change in the template language to allow logic into labels.

**Comment by lap42** [comment]
Perhaps this approach is easier to implement:

different tempates with the possibility to store different target folders or file names for Obsidian.

Within Obsidian, one could then filter which version is to be used further.

**Comment by tim-dalkiewicz** [comment]
Does somebody has started an implementation for this issue? 
If not, I would do so.
